Dockerfile

```diff
@@ -1,23 +1,18 @@
 # Openhim Console Dockerfile for latest changes
-FROM ubuntu:16.04
+FROM node:14.17-alpine
+RUN apk upgrade --update-cache --available
+RUN apk add git
 
-WORKDIR /etc/
+RUN mkdir -p /app
+WORKDIR /app
 
-# Update apt-repo list and install prerequisits
-RUN apt-get update
-RUN apt-get install -y git
-RUN apt-get install -y bzip2
-RUN apt-get install -y curl
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
-RUN apt-get install -y nodejs
-
-# Clone Openhim-console repo
-RUN git clone https://github.com/jembi/openhim-console.git
-
-WORKDIR /etc/openhim-console
+COPY package.json /app
```

Comment:
Nginx will serve html files in this dir `/usr/share/nginx/html`

Reply:
Yes, just run `npm install` and that will take care of everything

Reply:
cool

Reply:
Why is this step separate from `COPY . .` ?

Reply:
forgot to remove it. Should be gone now.